lang/LangPrimSource/SC_ComPort.cpp

```diff
@@ -205,8 +205,25 @@ UDP::UDP(int inPortNum, HandlerType handlerType, int portsToCheck): mPortNum(inP
         }
     }
 
-    boost::asio::socket_base::send_buffer_size option(65536);
-    mUdpSocket.set_option(option);
+    boost::asio::socket_base::send_buffer_size send_buffer_size;
```

Comment:
nitpick, but we generally use camelCase, not snake case, i.e. `sendBufferSize` instead of `send_buffer_size`. Same for other variables.